layouts/_default/list.html

```diff
@@ -31,7 +31,11 @@
 
 {{ $paginator := .Paginate (where .Data.Pages "Type" "post") }}
 {{ range $paginator.Pages }}
+{{ if (ne ($.Param "truncate") false) }}
```

Comment:
You're right. This doesn't work—something about Page params not being available in the `range` context, or us not accessing them correctly. I don't have time to look at this more right now, but also, on second thought, I think it's better to just keep this simple and keep it as one site-wide setting. Sorry for making you do the extra work, but maybe we should just merge #59 instead.